ansible_collections/arista/avd/molecule/eos_cli_config_gen/documentation/devices/host1.md

```diff
@@ -4944,7 +4944,7 @@ interface Ethernet81/4
    traffic-engineering administrative-group 4,7-100,testgrp
    traffic-engineering srlg 16
    traffic-engineering metric 2
-   traffic-engineering min-delay static 2 milliseconds
+   traffic-engineering min-delay dynamic twamp-light fallback 2 milliseconds
```

Comment:
I am not seeing a table change for the ethernet interface. Check for typos in the doc template.